### DISCUSSION TITLE
Create a way to use the functions through the haxball app

### DISCUSSION THREAD

**Discussion by eduardonslv7** [wxyz-abcd/node-haxball#75] [Ideas]
I believe that many players choose to play through the [haxball app,](https://github.com/oghb/haxball-client) as it offers better fps and gameplay.
Would it be possible to create a program with the functions to use them in the haxball app?

**Comment by wxyz-abcd** [comment]
Of course it is possible. I already did that with very early versions like v0.5 or something for faster development, but now it is probably very outdated. I would have to almost write it from scratch.
